frontend/src/components/managePermissions/ManagePermissions.vue

```diff
@@ -68,10 +66,6 @@ const selectApplication = async (e: DropdownChangeEvent) => {
     }
 };
 
-const selectApplicationOptions = computed(() => {
```

Comment:
I feel there is no need to have this computed variable, can use the applicationsUserAdministers directly

Reply:
Sounds good. Could you also remove the unused **"computed" import** on line 2?
I am a bit curious that in original code at line 64 with the same variable `applicationsUserAdministers` used in if condition, **does it work** that way in javascript for Vue's `ref`? Or should it be `if (applicationsUserAdministers.value) {`? Maybe it does not even need this "if" condition... Would you mind to verify this condition?

Reply:
yeah, this is good catch, and it should check if it has a value and if the length is > 0, which means it should have at least one role, I'll change it
```
if (
        applicationsUserAdministers.value &&
        applicationsUserAdministers.value.length > 0
    )
```